test/functionalTest/cases/1308_subscription_cache/cache_reset_from_db0_24_0.test

```diff
@@ -0,0 +1,394 @@
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Possible Bug in Subscription Cache reloading from database for 0.24.0
+
+--SHELL-INIT--
+dbInit CB
+dbInit ftest-t1
+dbInit ftest-t2
+brokerStart CB 0 IPv4 -multiservice -subCacheIval 2
+
+--SHELL--
+
+#
+# 01. Create subscription for s0.* entities, attribute A, w/o tenant
+# 02. Create subscription for s1.* entities, attribute A, for tenant t1
+# 03. Create subscription for s2.* entities, attribute A, for tenant t2
+# 04. See subscriptions w/o tenant 
+# 05. See subscriptions for tenant t1
+# 06. See subscriptions for tenant t2
+# 07. Wait 3 secs so that the subscription cache has been reset from DB
+# 08. See subscriptions w/o tenant 
+# 09. See subscriptions for tenant t1
+# 10. See subscriptions for tenant t2
+#
+
+
+echo "01. Create subscription for s0.* entities, attribute A, w/o tenant"
+echo "=================================================================="
+payload='{
+  "entities": [
+    {
+      "isPattern": "true",
+      "id": "s0.*"
+    }
+  ],
+  "reference": "127.0.0.1",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [ "A" ]
+    }
+  ]
+}'
+orionCurl --url /v1/subscribeContext --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Create subscription for s1.* entities, attribute A, for tenant t1"
+echo "====================================================================="
+payload='{
+  "entities": [
+    {
+      "isPattern": "true",
+      "id": "s1.*"
+    }
+  ],
+  "reference": "127.0.0.1",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [ "A" ]
+    }
+  ]
+}'
+orionCurl --url /v1/subscribeContext --payload "$payload" --json --tenant t1
+echo
+echo
+
+
+echo "03. Create subscription for s2.* entities, attribute A, for tenant t2"
+echo "====================================================================="
+payload='{
+  "entities": [
+    {
+      "isPattern": "true",
+      "id": "s2.*"
+    }
+  ],
+  "reference": "127.0.0.1",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [ "A" ]
+    }
+  ]
+}'
+orionCurl --url /v1/subscribeContext --payload "$payload" --json --tenant t2
+echo
+echo
+
+
+echo "04. See subscriptions w/o tenant"
+echo "================================"
+echo 'db.csubs.findOne()' | mongo ftest
```

Comment:
This test has been removed, it doesn't make any sense to check for what's in mongo when testing the sub-cache. 
But, removed in the next PR, the one where I actually spent some time looking at these tests for 1308